iptables/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
```

Comment:
Any reason not to use the latest named tag `3.6`? That way, in the unlikely event that something goes wrong in future versions of alpine, you won't be affected.